teku/src/test/resources/tech/pegasys/teku/cli/subcommand/spec_config.json

```diff
@@ -0,0 +1,97 @@
+{
+  "CONFIG_NAME": "mainnet",
+  "PRESET_BASE": "mainnet",
```

Comment:
we might name this better... something like `config_missing_network_fields.json` so that we know what it is